storm-core/src/storm.thrift

```diff
@@ -243,6 +243,16 @@ struct SubmitOptions {
   2: optional Credentials creds;
 }
 
+enum NumErrorsChoice {
+  ALL,
+  NONE,
+  ONE
+}
+
+struct GetInfoOptions {
+  1: optional NumErrorsChoice num_err_choice;
```

Comment:
Instead of an Enum don't you think a pagination struct with a start and end would be more flexible?